packages/x-license-pro/src/utils/licenseErrorMessageUtils.ts

```diff
@@ -14,9 +14,10 @@ function showError(message: string[]) {
 
 export function showInvalidLicenseError() {
   showError([
-    'MUI: Invalid license.',
+    'MUI: Invalid license key.',
```

Comment:
Nice, now it matches with https://mui.com/x/advanced-components/#validation-errors.